mesonbuild/dependencies/framework.py

```diff
@@ -70,12 +77,13 @@ def detect(self, name: str, paths: T.List[str]) -> None:
             self.is_found = True
             return
 
-    def _get_framework_path(self, path: str, name: str) -> T.Optional[Path]:
+    def _get_framework_path(self, path: str, name: str) -> T.Optional[tuple[Path, str]]:
```

Comment:
I think you should either rename the function to `_get_framework_path_and_name`, **or** return a single path like `d / framework_name` and use `path.name` in the caller

Reply:
I ended up using `path.stem` on the returned path.